tests/benchmarks/current_consumption/uart_idle/testcase.yaml

```diff
@@ -14,3 +14,16 @@ tests:
       pytest_root:
         - "${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py::test_measure_and_data_dump_power_consumption_with_rx_wakeup_54L_uart_idle"
     timeout: 80
+  benchmarks.current_consumption.uart_idle_uart30:
+    platform_allow:
+      - nrf54l15dk/nrf54l15/cpuapp
+    integration_platforms:
+      - nrf54l15dk/nrf54l15/cpuapp
+    extra_args:
+      - DTC_OVERLAY_FILE="boards/nrf54l15dk_nrf54l15_cpuapp_uart30.overlay"
+    harness: pytest
+    harness_config:
+      fixture: ppk_power_measure
+      pytest_root:
+        - "${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py::test_measure_and_data_dump_power_consumption_with_rx_wakeup_54L_uart_idle"
```

Comment:
That needs to be different function as idle current is different.

Reply:
Not much really, it still fist within the specified limit.

Reply:
In this test we want to track this lower value.